live/root/usr/lib/dracut/modules.d/99agama-cmdline/agama-network-compat.sh

```diff
@@ -0,0 +1,128 @@
+#!/bin/bash
+
+[ -e /dracut-state.sh ] && . /dracut-state.sh
+
+. /lib/dracut-lib.sh
+. /lib/net-lib.sh
+
+ifcfg_to_ip() {
+  local ip
+  local v="${2}",
+  local interface="$1"
+  local conf_path="/etc/cmdline.d/40-agama-network.conf"
+  set --
+  while [ -n "$v" ]; do
+    set -- "$@" "${v%%,*}"
+    v=${v#*,}
+  done
+
+  if [[ $# -eq 0 ]]; then
+    echo "IFCFG 0 options given, must be wrong"
```

Comment:
the text looks a bit strange..I would write something like "no parameters given to ifcfg option. Skipping" ( or what action happen after return 1

Reply:
was adde mainly for debugging we could even skip it.. or modify as suggested